packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -319,17 +319,32 @@ def test_dcos_diagnostics_report(dcos_api_session):
         assert len(report_response['Nodes']) > 0
```

Comment:
You need to add this test in a test group in `test_groups.yaml`.

Reply:
Why. The test is running in group 1

Reply:
Ah shit. It seemed to me that the test is new but it's not. GitHub fooled me here.